01-HTML-CSS-Basics/1-HTML/Activity3.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+	<meta name="viewport" content="width=device-width, user-scalable=no">
+    <meta http-equiv="refresh" content="50" />
+    <meta name="author" content="Santiago Garzon" />
+    
+    <title>My expenses</title>
+  </head>
+  <body>
+    <h1>My expenses</h1>
+    	<p></p>
+	    <table border="1" style="width:100%">
+		  <tr>
+		    <th>Description</th>
```

Comment:
A good practice is to put the th tags into a thead tag. The rest can be inside a tbody tag.